CMakeLists.txt

```diff
@@ -25,6 +25,15 @@ include(os_release_info.cmake)
 
 get_os_release_info(os_name os_version)
 
+if (CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT OR CMAKE_INSTALL_PREFIX STREQUAL "")
+    set(CMAKE_INSTALL_PREFIX "/usr" CACHE PATH "..." FORCE)
+endif()
+message(STATUS "CMAKE_INSTALL_PREFIX = ${CMAKE_INSTALL_PREFIX}")
+
+if(NOT DEFINED MEDIA_VERSION OR MEDIA_VERSION STREQUAL "")
+    set(MEDIA_VERSION "1.0.0")
```

Comment:
Can we have explicit setting of the version? I.e. version stored here or in the file which we read and which is updated regularly once the version should be bumped. I that way developers will always roughly understand with which version of the code they work.

Reply:
Is this location ok? And is this version string ok? Seems like previously, version wasn't defaulted and readme always stuck with '2.0.0'.

Reply:
/usr/local location is definitely ok.